### ISSUE TITLE
Support HTTP/2.0?

### ISSUE THREAD

**Issue by hxl-dy** [http-party/http-server#641]
**Steps to reproduce the issue, if applicable. Include the actual command and output and/or stack trace.**

**What did you expect to happen?**

**If the issue is a feature request, what is the motivation / use case for it?**

**Tell us about your environment**
- **exact http-server version:**
- **Node version:**
- **Platform:**

**Other information (related issues, suggestions for a fix, etc):**

**Comment by christjt** [comment]
Any plan / news for this? Since this package consumed ecstatic, I assume there are no dependencies that http-server is relying on to implement this?

**Comment by thornjad** [comment]
Yeah this should be internal to http-server now. I'd gladly take a look at a PR implementing this!

**Comment by Bruce-Hopkins** [comment]
I can implement this. Can I be assigned the issue?
